nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -199,8 +199,12 @@ void WaypointFollower::followWaypointsLogic(
     get_logger(), "Received follow waypoint request with %i waypoints.",
     static_cast<int>(poses.size()));
 
-  if (poses.size() == 0) {
-    action_server->succeeded_current(result);
+  if (poses.empty()) {
+    RCLCPP_ERROR(
+      get_logger(),
+      "Empty vector of Waypoints passed to waypoint following logic. "
```

Comment:
This warning isn't necessarily true. It could be that it failed to convert to GPS.

Reply:
If there is a failure during converting the GPS waypoints we already report that https://github.com/jediofgever/navigation2/blob/c28b93e25b9b0d1b9ff14d5f080597ca141a895c/nav2_waypoint_follower/src/waypoint_follower.cpp#L423
logically this warning is always true, though the reason  of passing empty vector is quite ambiguous.

Reply:
Perhaps `Empty vector of waypoints passed to waypoint following action potentially due to conversation failure or empty request.`

Just give a little more context in both so its clear that there's other reasons they could be seeing this